Source/SIMPLib/CoreFilters/ConvertColorToGrayScale.h

```diff
@@ -43,17 +42,18 @@
 /**
  * @brief The ConvertColorToGrayScale class. See [Filter documentation](@ref ConvertColorToGrayScale) for details.
  */
-class SIMPLib_EXPORT ConvertColorToGrayScale : public AbstractFilter
+class ConvertColorToGrayScale : public AbstractFilter
```

Comment:
SIMPLib_EXPORT is important for loading the class outside of the project it is defined in.  My guess is that there will either be compile errors when building DREAM.3D, or this filter will not be accessible inside of the GUI.

Reply:
I can't confirm if the filter showed up in the GUI but it built fine. For safety's sake I've re-included that snippet along with making sure to clang-format the files. I'll open up another PR with your included suggestion, thanks Matt!